spec/faraday/adapter/em_synchrony_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+RSpec.describe Faraday::Adapter::EMSynchrony do
+  features :request_body_on_query_methods, :reason_phrase_parse, :skip_response_body_on_head, :parallel
+
+  it_behaves_like 'an adapter'
+
+  it 'allows to provide adapter specific configs' do
+    url = URI('https://example.com:1234')
+    adapter = Faraday::Adapter::EMSynchrony.new nil, inactivity_timeout: 20
```

Comment:
```suggestion
    adapter = described_class.new nil, inactivity_timeout: 20
```

Reply:
Good spot, other adapters tests are affected as well, so I'll change it everywhere for consistency 😃

Reply:
If you're going to change it everywhere it should it be a separate pull request since it's not at all related to adding the em-synchrony test?

(For the record I prefer using the full class name rather than `described_class` but don't care enough to try and change anyone's mind about it. :man_shrugging: )

Reply:
I admit I could have done that in a separate PR 😄.
It's going to be only one line per test file, so if @olleolleolle don't mind I'd keep the change here rather than moving it into another PR.